exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/DrillFunctionRegistry.java

```diff
@@ -218,4 +302,141 @@ private void registerOperatorsWithoutInference(DrillOperatorTable operatorTable)
       }
     }
   }
+
+  /**
+   * Function registry holder. Stores function implementations by jar name, function name.
+   */
+  private class GenericRegistryHolder<T, U> {
```

Comment:
Similarly, U is a parameter class. GenericRegistryHolder is private, so the only uses of this class are in this file. The only use in this file specifies DrillFuncHolder as the value of U. So, code can be simper by just using that class name instead of a parameter.